projects/ngx-prx-styleguide/src/assets/styles/_pikaday-custom.scss

```diff
@@ -1,58 +1,83 @@
-
```

Comment:
I did a lot of work in this file. Converted hex values to variables, and ensure color contrast passes.